.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: scala
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION compile 'fastOptStage::testOnly T5514 scala.util.parsing.combinator.RegexParsersTest'
+  - sbt ++$TRAVIS_SCALA_VERSION compile 'testOnly T5514 scala.util.parsing.combinator.RegexParsersTest'
```

Comment:
Ach, almost. You should also put `set scalaJSStage in Global := FastOptStage` so that this is still tested in fastOpt mode. So that line should be:
```
 - sbt ++$TRAVIS_SCALA_VERSION 'set scalaJSStage in Global := FastOptStage' compile 'testOnly T5514 scala.util.parsing.combinator.RegexParsersTest'
```
Sorry, I should probably have foreseen this and told you directly.

Anyway, thanks for bearing with my review, I know they're very harsh ;-)